src/homeworks/test_48/index.ts

```diff
@@ -0,0 +1,23 @@
+type dataInput = {
+    year: number,
+    month: number, 
+    day: number
+}
+
+const outputDay= [
+    "Domenica",
+    "Lunedi",
+    "Martedi",
+    "Mercoledi",
+    "Giovedi",
+    "Venerdi",
+    "Sabato"
+]
+
+const getDay = ({year, month, day}: dataInput): string => {
+    const getIndex = new Date(year, month, day).getDay()
+    return outputDay[getIndex]
```

Comment:
Di solito si utlizza `get<???>` quando la variable e' una funzione. Questa e' una variable constante `number`. E' un `index`.
```suggestion
    const index = new Date(year, month, day).getDay()
    return outputDay[index]
```